viral/offline_sequence_detection.py

```
@@ -0,0 +1,940 @@
+    return np.clip(binned_positions, 0, n_bins - 1)
+
+
+def plot_confusion_matrix_actual_vs_decoded_position(
+    actual_position: np.ndarray,
+    decoded_position: np.ndarray,
+    session: Cached2pSession,
+    bayesian_config: BayesianDecodingConfig,
+    plot_landmarks: bool = True,
+) -> None:
+    landmarks_cm = [45, 90, 135]
+    landmarks = [l / bayesian_config.bin_size_spatial for l in landmarks_cm]
```

Ruff (0.14.5)

Ambiguous variable name: `l`

(E741)